src/software/backend/simulation/physics_simulator.cpp

```diff
@@ -0,0 +1,46 @@
+#include "software/backend/simulation/physics_simulator.h"
+
+#include <Box2D/Box2D.h>
+
+PhysicsSimulator::PhysicsSimulator(const World& world)
+{
+    b2Vec2 gravity(0, 0);
+    physics_world           = std::make_shared<b2World>(gravity);
+    physics_world_timestamp = Timestamp::fromSeconds(0);
+
+    // 5 and 8 here are somewhat arbitrary values for the velocity and position
+    // iterations but are the recommended defaults from
+    // https://www.iforce2d.net/b2dtut/worlds
+    velocity_iterations = 8;
+    position_iterations = 3;
```

Comment:
can these be constants in physics_simulator.h?

Reply:
Also, the comment is _already_ out of date with the code (the values are different), so perhaps let's make it a bit more generic

Reply:
whoops I think that was just a typo the first time I wrote the comment. Made these constants in the header file.